pcdsdevices/pv_positioner.py

```diff
@@ -160,3 +162,76 @@ def _setup_move(self, position):
     def _toggle_done(self):
         self.done.put(0, force=True)
         self.done.put(1, force=True)
+
+
+class PVPositionerNoInterrupt(PVPositioner):
+    """
+    A PV positioner whose moves cannot be interrupted.
+
+    If we try to start a new move before the previous move completes,
+    instead we will get a clear error advising us to wait.
+
+    Parameters
+    ----------
+    prefix : str, optional
+        The device prefix used for all sub-positioners. This is optional as it
+        may be desirable to specify full PV names for PVPositioners.
+    limits : 2-element sequence, optional
+        (low_limit, high_limit)
+    name : str
+        The device name
+    egu : str, optional
+        The engineering units (EGU) for the position
+    settle_time : float, optional
+        The amount of time to wait after moves to report status completion
+    timeout : float, optional
+        The default timeout to use for motion requests, in seconds.
```

Comment:
The parent `PVPositioner` lists the signals needed to properly subclass, is it worth repeating those here?  Or do we expect people to know to look at `PVPositioner`'s docstring?

Reply:
It's a good question, I went and copied only about half of the parent's docstring. Maybe it's worth copying the rest since I went through the trouble at all.

Reply:
Added back in https://github.com/pcdshub/pcdsdevices/pull/1183/commits/c0b4fbfd5b253127e35f8d80739d43b8262d6e49